plugins/ccxtExchange_test.go

```diff
@@ -14,6 +14,8 @@ import (
 
 var supportedExchanges = []string{"binance"}
 var emptyAPIKey = api.ExchangeAPIKey{}
+var emptyParams = api.CcxtParam{}
+var emptyHeader = api.ExchangeHeader{}
```

Comment:
@Reidmcc  there are a few tests here (`ccxtExchange_test.go`) that include the list of "tested" exchanges (example: `TestGetOrderConstraints_Ccxt_Precision`). Can you add coinbasepro to that test?